Fix compilation error on Solaris OS

./deps/v8/src/base/platform/platform-posix.cc:73:16: error: conflicting declara
tion of C function 'int madvise(caddr_t, std::size_t, int)'
 extern "C" int madvise(caddr_t, size_t, int);
                ^~~~~~~
In file included from ../deps/v8/src/base/platform/platform-posix.cc:20:0:
/usr/include/sys/mman.h:232:12: note: previous declaration 'int madvise(void*, std::size_t, int)'
 extern int madvise(void *, size_t, int);
            ^~~~~~~

Only in node-10.16.2: .patched
Only in node-10.16.2: .patched-001-madvise.patch
diff -ru node-10.16.2.orig/deps/v8/src/base/platform/platform-posix.cc node-10.16.2/deps/v8/src/base/platform/platform-posix.cc
--- node-10.16.2.orig/deps/v8/src/base/platform/platform-posix.cc	2019-08-22 19:19:01.037592472 +0000
+++ node-10.16.2/deps/v8/src/base/platform/platform-posix.cc	2019-08-22 19:28:47.689256762 +0000
@@ -68,14 +68,6 @@
 #define MAP_ANONYMOUS MAP_ANON
 #endif
 
-#if defined(V8_OS_SOLARIS)
-#if (defined(_POSIX_C_SOURCE) && _POSIX_C_SOURCE > 2) || defined(__EXTENSIONS__)
-extern "C" int madvise(caddr_t, size_t, int);
-#else
-extern int madvise(caddr_t, size_t, int);
-#endif
-#endif
-
 #ifndef MADV_FREE
 #define MADV_FREE MADV_DONTNEED
 #endif
Only in node-10.16.2/deps/v8/src/base/platform: platform-posix.cc.~1~
